BotProject/Templates/CSharp/appsettings.json

```diff
@@ -18,7 +17,7 @@
   },
   "telemetry": {
     "logPersonalInformation": false,
-    "logActivities": true
+    "logActivities": false
```

Comment:
I think we should set this to true.  This is the default in the SDK and activities will only be logged once someone populates their InstrumentationKey.  We already set LogPersonalInformation to false by default which I think is enough.

Reply:
fixed